### ISSUE TITLE
508 - Make 18f logo a proper image with alt text

### ISSUE THREAD

**Issue by nickbristow** [18F/microsite-template-18f#3]
18f logo link is currently done with CSS. It should be a regular image with alt text = '18f logo'
